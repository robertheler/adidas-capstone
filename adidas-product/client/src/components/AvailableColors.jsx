import React, { Component } from 'react';
import styled from 'styled-components';

const StyledHeader = styled.h1`
  font-size: 20px;
  font-family: AdineuePRO,Helvetica;
  font-weight: 600;
  margin: 0 0 7px 0;
  word-wrap: normal;
  text-transform: uppercase;
`;

class AvailableColors extends Component {
  constructor(props){
    super(props);
  }

  render() {
    return (
        <StyledHeader>Available colors</StyledHeader>
    );
  }
}

export default AvailableColors;