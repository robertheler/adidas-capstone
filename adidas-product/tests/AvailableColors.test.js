import React from 'react';
import { shallow, mount, render } from 'enzyme';
import AvailableColors from '../client/src/components/AvailableColors';

// json data will be used to generate props
var data = require('../db/data.json');

function setup() {
  const props = {
    color: 0,
    product: data[0]
  };
  const wrapper = shallow(<AvailableColors color={props.color} product={props.product} isSelected={props.isSelected} handleColorChange={props.handleColorChange}/>);
  return { wrapper, props };
}

describe('Title Component Test Suite', () => {
  const { wrapper, props } = setup();

  it('It should render', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('It should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});