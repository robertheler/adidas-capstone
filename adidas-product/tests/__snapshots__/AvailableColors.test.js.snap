// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component Test Suite It should render correctly 1`] = `
<styled.h1>
  Available colors
</styled.h1>
`;
